static/templates/subscription.hbs

```diff
@@ -2,7 +2,7 @@
 {{#with this}}
 <div class="stream-row" data-stream-id="{{stream_id}}" data-stream-name="{{name}}">
 
-    <div class="check {{#if subscribed }}checked{{/if}} sub_unsub_button {{#unless should_display_subscription_button}}disabled{{/unless}}">
+    <div class="check {{#if subscribed }}checked{{/if}} sub_unsub_button {{#unless should_display_subscription_button}}disabled{{/unless}} tippy-zulip-tooltip place-top-subunsub" data-tippy-content="{{#if subscribed }}{{t "Click to unsubscribe stream" }}{{else}}{{t "Subscribe to stream" }}{{/if}}">
```

Comment:
You can add `data-tippy-placement="top"` here and get rid of additional delegate function.

Reply:
thanks for the help!